util/install.py

```diff
@@ -83,6 +87,49 @@ def install_quagga():
         break
 
 
+def install_openr(openr_release=OpenrRelease,
+                  openr_remote="https://github.com/facebook/openr.git"):
+    openr_install = os.path.join(args.output_dir, "openr")
+    openr_build = os.path.join(openr_install, "build")
+    openr_buildscript = os.path.join(openr_build, "build_openr_debian.sh")
+    debian_system_builder = "debian_system_builder/debian_system_builder.py"
+    sh("git clone %s" % openr_remote,
```

Comment:
It would be nice to install git before cloning of the openr repository.

```suggestion
    dist.install("git")
    sh("git clone %s" % openr_remote,
```

Reply:
:+1: I added it to the beginning of the function